tests/Dockerfile

```diff
@@ -1,4 +1,21 @@
-FROM autopilotpattern/testing
-COPY tests/tests.py /src/
-COPY *.yml /src/
-COPY setup.sh /src/
+FROM drydock/u14pyt:prod
+
+RUN apt-get update \
+    && apt-get install -y apt-transport-https ca-certificates \
+    && echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list \
+    && apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
+    && apt-get update \
+    && apt-get install -y nodejs git docker-engine \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN npm install -g triton json
+
+RUN git clone https://github.com/autopilotpattern/testing.git \
+    && cd testing \
+    && git checkout python3 \
```

Comment:
I'm going to remove the `git checkout python3` line once that branch is merged, but good call on the rest.